src/components/Utils.js

```diff
@@ -0,0 +1,27 @@
+function populateDeck() {
+  const CARDS = ['A', 2, 3, 4, 5, 6, 7, 8, 9, 10, 'J', 'Q', 'K'];
+  const SUITS = ['hearts', 'spades', 'diamonds', 'clubs'];
```

Comment:
Essas 2 constantes são locais à função. Toda vez que essa função for executada os arrays serão alocados novamente na memória. Elas podem ser declaradas no nível do módulo para evitar isso.